Userland/Applications/DisplaySettings/BackgroundSettingsWidget.h

```diff
@@ -23,20 +23,20 @@ class BackgroundSettingsWidget : public GUI::SettingsWindow::Tab {
     C_OBJECT_ABSTRACT(BackgroundSettingsWidget);
 
 public:
-    static ErrorOr<NonnullRefPtr<BackgroundSettingsWidget>> try_create(bool& background_settings_changed);
+    static ErrorOr<NonnullRefPtr<BackgroundSettingsWidget>> try_create();
+    static ErrorOr<NonnullRefPtr<BackgroundSettingsWidget>> try_create(bool* background_settings_changed);
+
     virtual ~BackgroundSettingsWidget() override = default;
 
     virtual void apply_settings() override;
 
 private:
-    BackgroundSettingsWidget(bool& background_settings_changed);
-
     ErrorOr<void> create_frame();
     ErrorOr<void> load_current_settings();
 
     Vector<String> m_modes;
 
-    bool& m_background_settings_changed;
+    bool* m_background_settings_changed;
```

Comment:
This should stay as a reference.

Reply:
GMLCompiler requires the class to have the default constructor available, but that's impossible if we have a reference member (since we'd have to initialize it to _something_ in the constructor). What should I do about this?
![image](https://github.com/SerenityOS/serenity/assets/51789606/d5d4ab0b-189f-4363-9904-5951d589fd09)

Reply:
Ah, I see. We probably want to make some GML-side changes for this then before continuing, so we may want to drop this commit then for now.

Reply:
Got it. Also what do you mean by "drop" the commit? Should I get rid of it from history through git rebase?

Reply:
Also could I possibly contribute to that as well, if a new issue regarding this was opened? :)

Reply:
Yes, making a backup branch (as to not lose the work completely) and then rebasing away the commit so that it's no longer in the PR.